nexussdk/storages.py

```diff
@@ -248,36 +270,42 @@ def update_s3_storage(self, org_label: str, project_label: str,
         if secret_key is not None:
             payload["secretKey"] = secret_key
 
+        if max_file_size is not None:
+            payload["maxFileSize"] = max_file_size
+
         return self.update_(org_label, project_label, payload, storage_id, rev)
 
-    def update_external_disk_storage(self, org_label: str, project_label: str, endpoint: str, folder: str,
-                                     storage_id: str, rev: int, read_permission: Optional[str] = None,
-                                     write_permission: Optional[str] = None, default: bool = False,
-                                     credentials: Optional[str] = None) -> Dict:
-        """Update external disk storage.
+    def update_remote_disk_storage(self, org_label: str, project_label: str, folder: str,
+                                   storage_id: str, rev: int, endpoint: Optional[str] = None,
+                                   read_permission: Optional[str] = None, write_permission: Optional[str] = None,
+                                   default: bool = False, credentials: Optional[str] = None,
+                                   max_file_size: Optional[int] = None) -> Dict:
+        """Update remote disk storage.
     
         :param org_label: Label of the organization the storage belongs to.
         :param project_label: Label of the project the storage belongs to.
-        :param endpoint: endpoint to communicate with the external storage
-        :param folder:  external storage folder (similar concept to bucket in the S3)
+        :param folder: remote storage folder (similar concept to bucket in the S3)
         :param storage_id: the storage ID
         :param rev: last known revision of the storage
+        :param endpoint: (optional)endpoint to communicate with the remote storage
         :param read_permission: (optional) the permission required in order to download a file from this storage
         :param write_permission: (optional) the permission required in order to upload a file to this storage
         :param default: (optional) whether the storage should be the default storage for the project, defaults to False
-        :param credentials: (optional) external storage optional Bearer Token
+        :param credentials: (optional) remote storage optional Bearer Token
         :return: The Nexus metadata of the updated storage.
         """
         payload = {
-            "@type": "nxv:ExternalDiskStorage",
-            "endpoint": endpoint,
+            "@type": "nxv:RemoteDiskStorage",
```

Comment:
RemoteDiskStorage instead of nxv:RemoteDiskStorage

Reply:
Done